src/coreclr/src/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -96,10 +98,6 @@
 
   <!-- Assembly attributes -->
   <PropertyGroup>
-    <AssemblyName>System.Private.CoreLib</AssemblyName>
-    <AssemblyVersion>5.0.0.0</AssemblyVersion>
-    <ExcludeAssemblyInfoPartialFile>true</ExcludeAssemblyInfoPartialFile>
-    <GenerateTargetFrameworkAttribute>false</GenerateTargetFrameworkAttribute>
```

Comment:
Why do we need to generate the target framework atrribute in CoreLib? It is just an extra cruft that nobody cares about.

Reply:
We generate the attribute for all assemblies in libraries, even for Private ones like System.Private.Xml. Mainly I did this to follow-up on your comment here: https://github.com/dotnet/corefx/pull/32680#issuecomment-429478625

Reply:
Ah, ok - I forgot about that discussion. As you can tell, I still wonder why we are generating this cruft.

Reply:
> I still wonder why we are generating this cruft for inbox assemblies.

It's funny because in the past you mentioned that we should enable it for all - even the inbox ones. I don't have a strong opinion here but I would like to enable this for the sake of consistency. Also I prefer not opting out of the default sdk behavior if there isn't a strong reason.

Also as a side note, I noticed partners taking a dependency on the attribute, ie vstest for test assemblies: https://github.com/microsoft/vstest/blob/7b6248203164f8ea821f6795632bd22e0b69afb0/src/Microsoft.TestPlatform.ObjectModel/Utilities/AssemblyLoadWorker.cs#L67.

Reply:
> It's funny because in the past you mentioned that we should enable it for all - even the inbox ones.

I do not think that I have said that. I [have said](https://github.com/dotnet/corefx/pull/32680#issuecomment-429478625) that if all inbox ones have it, CoreLib should have it too.

>  vstest for test assemblies:

These are not inbox assemblies.

Reply:
> I have said that if all inbox ones have it, CoreLib should have it too.

Thanks for clarifying.

> And I believe that this path is .NET Framework specific.

I know there was another code path somewhere for .NET Core but I can't find it right now...

Reply:
If you think we should remove the property from all inbox assemblies we can do that in a follow-up PR.